Octo-Bot/packet-generator-bot/test.sh

```diff
@@ -0,0 +1,15 @@
+chmod a+x *.sh &&
```

Comment:
Can you a little documentation for this function?
What are the expected results?